deps/chakrashim/src/v8functiontemplate.cc

```diff
@@ -264,12 +264,14 @@ class FunctionTemplateData : public TemplateData {
             return nullptr;
           }
         }
-
-        if (JsSetProperty(*prototype,
-                          iso->GetCachedPropertyIdRef(
-                            jsrt::CachedPropertyIdRef::constructor),
-                          function, false) != JsNoError ||
-            JsSetProperty(function,
+        if (jsrt::DefineProperty(*prototype,
+            iso->GetCachedPropertyIdRef(
+                jsrt::CachedPropertyIdRef::constructor),
+            jsrt::PropertyDescriptorOptionValues::True,
```

Comment:
Nit: can you comment the individual bool params so its easier to read which properties are being set on the property descriptor without having to refer to the definition of jsrt::DefineProperty?